docs/source/func.batch_norm.rst

```diff
@@ -0,0 +1,58 @@
+Patching Batch Norm
+===================
+
+What's happening?
+-----------------
+Batch Norm requires in-place updates to running_mean and running_var of the same size as the input.
+Functorch does not support inplace update to a regular tensor that takes in a batched tensor (i.e.
+``regular.add_(batched)`` is not allowed). So when vmaping over a batch of inputs to a single module,
+we end up with this error
+
+How to fix
+----------
+All of these options assume that you don't need running stats. If you're using a module this means
+that it's assumed you won't use batch norm in evaluation mode. If you have a use case that involves
+running batch norm with vmap in evaluation mode, please file an issue
```

Comment:
Are we not going to recommend replacing the batch norm with another normalization layer? That should be good for completeness so the user knows their options

Reply:
Added language to 1 + 2 on how to replace with GroupNorm. From Opacus' work, generally replacing the norm layer with a different layer is fragile but the version functorch offers (`replace_batch_norm_modules_`) turns off tracking should work since it just updates the layer in place instead of trying to replace the whole layer